### DISCUSSION TITLE
Qt Creator setup for building and debugging

### DISCUSSION THREAD

**Discussion by MattiaMontanari** [fougue/mayo#300] [Q&A] [answered]
I read in #226 that, in older to debug and build Mayo from Qt Creator, Qt Creator should be launched from a folder that looks like the installation folder of OCC. Is that correct? Is there a better way to achieve this? For example, in Visual Studio you could point at the installation path of OCC and include it in the project dependencies.

I'm on MacOS

**Comment by HuguesDelorme** [comment]
Hello @MattiaMontanari
No this is no longer recommanded since Mayo is using cmake(see https://github.com/fougue/mayo/issues/53)
You can just load Mayo in QtCreator. The CMake script will probably find OpenCascade and Qt.
If you have troubles feel free to ask for help in this thread

**Comment by MattiaMontanari** [comment, marked as answer]
@HuguesDelorme great! I didn't know QT Creator could digest CMakeLists files. Works just fine, thanks! This repo however does have a `.pro` file and I wonder what's its purpose. I have opened that one initially that Qt Creator..

**Reply by HuguesDelorme** [reply]
Yes there's still the QMake project file `scripts/qmake/mayo.pro` but it shouldn't be used(this has to be removed in the short-term)
The root `CMakeLists.txt` file must be used to build Mayo
